dotnet/src/webdriver/SeleniumManager.cs

```diff
@@ -36,27 +36,28 @@ public static class SeleniumManager
     {
         private static readonly ILogger _logger = Log.GetLogger(typeof(SeleniumManager));
 
-        private static readonly string BinaryFullPath = Environment.GetEnvironmentVariable("SE_MANAGER_PATH");
+        private static string? _binaryFullPath;
+        private static string BinaryFullPath => _binaryFullPath ??= GetBinaryFullPath();
```

Comment:
Cunning line of code. If we would use `Lazy<string> _lazyBinaryFullPath`, then it would look more straightforward and simple.

```csharp
private static Lazy<string> _lazyBinaryFullPath = new Lazy<string>(...);
private static string BinaryFullPath => _lazyBinaryFullPath.Value;
```

And now seems we don't need `field` and `property` with identical purposes, just use `_lazyBinaryFullPath.Value` everywhere.

Reply:
Good call. PR has been updated.